code/modules/mob/living/silicon/robot/drone/drone.dm

```diff
@@ -13,7 +13,8 @@
 	braintype = "Robot"
 	lawupdate = 0
 	density = 0
-	req_access = list(access_engine, access_robotics)
+	req_access = null
```

Comment:
This statement should be deleted if it is being replaced.

Reply:
req_access is by default list(access_robotics), so it needs to be nulled or engineers won't be able to reboot the drone. There might be a better way but I'm leery of substantially altering the structure of the robot type for the sake of a single edge case.